cli/runner.go

```diff
@@ -27,45 +29,18 @@ import (
 	"golang.org/x/net/context"
 )
 
-func init() {
-	usageTemplate := template.Must(template.New("usage").Parse(
-		`{{.Program}} runs the Evergreen processes at regular intervals.
-
-Usage:
-  {{.Program}} [flags] [process name]
-
-Supported flags are:
-{{.FlagDefaults}}
-Supported proccesses are:{{range .Runners}}
-  {{.Name}}: {{.Description}}{{end}}
-
-Pass a single process name to run that process once,
-or leave [process name] blank to run all processes
-at regular intervals.
-`))
-
-	flag.Usage = func() {
-		// capture the default flag output for use in the template
-		flagDefaults := &bytes.Buffer{}
-		flag.CommandLine.SetOutput(flagDefaults)
-		flag.CommandLine.PrintDefaults()
-
-		// execute the usage template
-		usageTemplate.Execute(os.Stderr, struct {
-			Program      string
-			FlagDefaults string
-			Runners      []ProcessRunner
-		}{os.Args[0], flagDefaults.String(), Runners})
-	}
-}
-
 const (
 	schedulerInterval  = 20 * time.Second
 	hostinitInterval   = 20 * time.Second
 	defaultRunInterval = 60 * time.Second
 )
 
-func main() {
+type ServiceRunnerCommand struct {
+	ConfigPath string `long:"conf" default:"/etc/mci_settings.yml" description:"path to the service configuration file"`
```

Comment:
This doesn't seem to be used at all? Also could you make "conf" the short flag and "config" the long version?

Johns-MacBook-Pro:evergreen johnliu$ ./clients/darwin_amd64/evergreen service runner --conf "config/config.yml"
Error reading config file: open /etc/mci_settings.yml: no such file or directory